spinedb_api/helpers.py

```diff
@@ -911,3 +951,15 @@ def string_to_bool(string):
     if string in _FALSES:
         return False
     raise ValueError(string)
+
+
+class DisplayStatus:
```

Comment:
I tried using the enum.Enum directly instead of the `*DisplayStatus.values()` hack. couldn't get it to work, don't know if the problem is me or if there are some issues with Enum in SQLAlchemy version 1.3. I could try the \_\_members__ method with enum.Enum instead of using it straight in the SQLAlchemys Enum class though.

Reply:
I would not be surprised if Python's standard `Enum` would not work as a drop-in replacement to SqlAlchemy's own `Enum` --- they probably need their own specific functionality. Nevertheless, we could still consider using Python's `Enum` for `spinedb_api`.